.docs/README.md

```diff
@@ -54,13 +54,11 @@ extensions:
     orm.annotations: Nettrine\ORM\DI\OrmAnnotationsExtension
 
 orm.annotations:
-    paths:
+    mapping:
         # your app entities
-        - App/Model/Database/Entity
+        App\Model\Database\Entity: '%appDir%/Model/Database/Entity'
         # doctrine extensions entities
-        - Gedmo\Loggable\Entity
-        - Gedmo\Loggable\Entity
-        - Gedmo\Tree\Entity
+        Gedmo: '%appDir%/../vendor/gedmo'
```

Comment:
nefunguje můj nápada @f3l1x nevíš, jak to fixnout?

Reply:
Jaky napad myslis? A btw, nemusis to obalovat do `'`. ;-)

Reply:
exception jsem ti poslal na slack

Reply:
našel jsem řešení `Gedmo\Translatable: '%appDir%/../vendor/gedmo/doctrine-extensions/lib/Gedmo/Translatable/Entity'`

Reply:
opraveno v b40e4a3dc5b92ba137fc27fae4c0d82fb5c863df